app/assets/stylesheets/components/_section.scss

```diff
@@ -1,51 +1,91 @@
-.dps-login-section {
-  @include padding($site-margins-mobile);
-  background-color: $color-gray-lightest;
-
-  .usa-layout-docs-sidenav & {
-    @include margin($site-margins null);
-    @include padding($site-margins 0);
-    background-color: transparent;
-    border-top: 0.25rem solid $color-primary-darkest;
-  }
+.homepage-section {
+  @include padding($site-margins-mobile null);
 
-  p:first-child {
-    margin-top: 0;
+  @include media($medium-screen) {
+    @include padding($site-margins null);
   }
 
-  .usa-button {
-    display: block;
-    margin: 0;
+  & + & {
+    border-top: 2px dotted $color-gray-cool-light;
   }
-}
 
-.homepage-section {
-  @include padding($site-margins null);
-  border-bottom: 1px solid $color-gray-light;
+  img {
+    width: 4rem;
 
-  &:last-of-type {
-    border-bottom: none;
+    @include media($nav-width) {
+      margin-right: 3rem;
+      width: 9rem;
+    }
   }
 
-  img {
-    display: none;
-    margin-right: $site-margins;
-    width: 14rem;
+  ul {
+    clear: left;
+    margin-top: 0;
+    padding-top: 2rem;
 
-    @include media($medium-screen) {
-      display: block;
+    @include media($nav-width) {
+      clear: none;
+      margin-left: 12rem;
+      padding-top: 2.5rem;
     }
   }
 
-  h3 {
-    margin: 0;
+  .homepage-section-header {
+    color: $color-primary-darker;
+    font-size: 2.5rem;
 
     & + p {
       margin-top: 0.25rem;
+      margin-bottom: 0;
+    }
+
+    @include media($nav-width) {
+      font-size: 3.5rem;
+      font-weight: $font-bold;
     }
   }
 }
 
+.helpful-links-section {
+  @include margin($site-margins-mobile null);
+  @include padding($site-margins);
+
+  h4 {
+    border-bottom: 1px solid $color-cool-blue;
+    color: $color-cool-blue;
+    margin-top: 1rem;
+    padding-bottom: 0.5rem;
+  }
+
+  background-color: $color-gray-cool-light;
+}
+
+.homepage-service-section {
+  border-top: 0.25rem solid $color-primary-darkest;
+  h3 {
+    color: $color-primary-darkest;
+    font-family: $font-sans;
+    margin-bottom: 0;
+  }
+  p {
+    margin: 0;
+  }
+}
+
+.homepage-anchor-section {
+  .homepage-anchor-title {
+    color: $color-primary-darker;
+    font-size: 2.5rem;
+    font-weight: $font-bold;
+    margin: 3.5rem 1.5rem 1rem 1.5rem;
+    text-align: center;
+  }
+
+  @include media($medium-screen) {
+    display: none;
+  }
```

Comment:
I tend to organize rulesets that apply to the parent element (`.homepage-anchor-section` in this case) above rulesets for child elements (`.homepage-anchor-title` in this case):

```scss
.homepage-anchor-section {
  @include media($medium-screen) {
    …
  }

  .homepage-anchor-title {
    …
  }
}
```